packages/casbin/README.md

```diff
@@ -0,0 +1,143 @@
+# Casbin Middleware for Hono
+
+This is a third-party [Casbin](https://casbin.org) middleware for [Hono](https://github.com/honojs/hono).
+
+This middleware can be used to enforce authorization policies defined using Casbin in your Hono routes.
+
+## Installation
+
+```bash
+npm i hono @hono/casbin casbin
+```
+
+## Configuration
+
+Before using the middleware, you must set up your Casbin model and policy files.
+
+For details on how to write authorization policies and other information, please refer to the [Casbin documentation](https://casbin.org/).
+
+### Example model.conf
+
+```conf
+[request_definition]
+r = sub, obj, act
+
+[policy_definition]
+p = sub, obj, act
+
+[role_definition]
+g = _, _
+
+[policy_effect]
+e = some(where (p.eft == allow))
+
+[matchers]
+m = r.sub == p.sub && r.obj == p.obj && r.act == p.act
+```
+
+### Example policy.csv
+
+```csv
+p, alice, /dataset1/*, *
```

Comment:
Is the wildcard `*` working correctly?

It does not work but changed like the following, it works well:

```csv
p, alice, /dataset1/test, GET
```

Reply:
Sorry, the way the Casbin model was written in the README was incorrect.
You need to write the model in a way that allows wildcards, as shown below. (I've fixed it.)

https://github.com/honojs/middleware/pull/676/commits/4749e9d499375406a4d83b1f169e87656df1278b